hedera-node/src/main/java/com/hedera/services/throttling/DeterministicThrottling.java

```diff
@@ -130,10 +157,32 @@ public void rebuildFor(ThrottleDefinitions defs) {
 
 		functionReqs = newFunctionReqs;
 		activeThrottles = newActiveThrottles;
-
+		applyGasConfig();
```

Comment:
Is there a reason to call `applyGasConfig()` here, since it no longer changes with file `0.0.123` updates?